tasks/14_clone-arr/clone-arr.js

```diff
@@ -0,0 +1,3 @@
+export function cloneArr(arr) {
+  return arr.map((item) => item);
```

Comment:
Doesn't it looks weird for you ?  .map has its own purpose. Could you check any additional way to clone array ?